tests/aws/awslambda/test_lambda_destinations.py

```diff
@@ -475,5 +493,135 @@ def _assert_event_count(count: int):
 #     ...  # TODO
 #
 #
-# class TestLambdaDestinationEventbridge:
-#     ...  # TODO
+class TestLambdaDestinationEventbridge:
+    EVENT_BRIDGE_STACK = "EventbridgeStack"
+    INPUT_FUNCTION_NAME = "InputFunc"
+    TRIGGERED_FUNCTION_NAME = "TriggeredFunc"
+    TEST_QUEUE_NAME = "TestQueueName"
```

Comment:
nit: I'd suggest using the autogenerated names here instead to avoid any conflicts

Reply:
@dominikschubert: you mean instead of the function-names defined below, eg. this part, right? 
```
  input_fn_name = f"input-fn-{short_uid()}"
  triggered_fn_name = f"triggered-fn-{short_uid()}"
```
The names defined here `TEST_QUEUE_NAME` etc are just used as identifier for the output variables.